### DISCUSSION TITLE
Announcing the Completion of the Conflux Documentation Improvement Survey and Commencement of FC Reward Distribution

### DISCUSSION THREAD

**Discussion by darwintree** [Conflux-Chain/conflux-documentation#407] [Announcements]
[中文版本](#中文版本) | [English Version](#english-version)

## 中文版本

尊敬的诸位Conflux社区开发者：

感谢大家积极参与我们于2023年12月发起的“Conflux文档改进调查”。我们很高兴地宣布，该问卷调查与回访工作已圆满结束。我们收到了众多宝贵的反馈和建议，这将极大地帮助我们优化和提升Conflux文档，更好地服务于开发者和普通用户。针对文档中涉及的各项反馈，我们也已经做出了改进，欢迎各位访问文档网站确认。

现在，我们很荣幸地通知各位，奖励发放工作已经启动。根据活动规则，提交**有效**问卷的用户将获得FC奖励。此外，为文档优化提供了关键性建议的参与者，领取FC奖励的数额将更多。领取奖励的流程为：1. 通过您在问卷中留下的github账户，授权领取POAP(Proof of Attendance Protocol)。2. 使用领取了 POAP 的账户，领取FC奖励。也可参考文末图文指南。

**POAP领取的截止时间为主网区块号218641462， 预计为 2024年02月28日 17:50:20 (UTC+08:00)**。领取POAP后任何时间都可以领取附加的FC奖励。

此外，基于问卷的反馈，我们也计划为文档贡献者直接提供经济激励。激励细则将于近日公开。我们已经在 Github Issues 中列出了一些适合完成的任务（https://github.com/Conflux-Chain/conflux-documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22 ）， 欢迎诸位参与完成。

Conflux文档维护团队对诸位的支持和贡献表示衷心的感谢。开发者的建议对我们至关重要，也是我们不断前进的动力。敬请期待我们即将推出的更多优化和更新！

Conflux文档维护团队
2024年2月2日

#### 使用 Github 账户授权领取 POAP

访问 [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app)。按照图中指示填入您的Core Space地址。点击"one-shot authorize"完成授权后点击"Do Mint"获取POAP。
<img width="1088" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/f255c4cb-0715-4c9e-b151-edd69041c68d">

> 如果提示无权限，请尝试在上方卡片中检查您是否具有权限。

#### 领取 FC

访问 [Core Space 管理页](https://dual-space-nft-frontend.vercel.app/coreManage)。连接您刚刚领取POAP的账户。之后选择POAP，点击"Claim xx FC"按钮。

<img width="1074" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/2936d164-13dd-4dbf-b850-9927e00c87ad">

## English Version

Dear Conflux Community Developers,

Thank you for your enthusiastic participation in our "Conflux Documentation Improvement Survey," launched in December 2023. We're excited to share that the survey and its subsequent follow-up activities have now been successfully concluded. Your valuable feedback and suggestions have been instrumental, enabling us to significantly enhance the Conflux documentation, thereby improving the experience for both developers and general users. We have already implemented improvements based on your feedback and invite you to review these updates on our documentation website.

We are also pleased to announce the commencement of the rewards distribution process. In line with the rules of the activity, participants who submitted **valid** questionnaires will be eligible for FC rewards. Additionally, those who provided key insights for the documentation's optimization will receive a higher amount of FC rewards. The reward claiming process is as follows: 1. Use the GitHub account provided in your survey to authorize and receive the POAP (Proof of Attendance Protocol). 2. Claim your FC rewards using the account that has received the POAP. For further guidance, please refer to the step-by-step guide provided at the end of this announcement.

**The deadline to claim POAP is block number 218641462 of mainnet, estimated as February 28th, 2024 17:50:20 (UTC+08:00).** You can claim the FC rewards any time only if you own the POAP.

In response to the survey feedback, we are also exploring the possibility of offering direct financial incentives to those who contribute significantly to our documentation. Details regarding this initiative will be made available shortly. We have already identified several tasks that are ripe for completion, which you can find in our Github Issues (https://github.com/Conflux-Chain/conflux-documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). Your involvement in these tasks would be greatly appreciated.

The Conflux Documentation Maintenance Team extends heartfelt thanks for your ongoing support and valuable contributions. Your insights are not only vital to us but also drive our continuous improvement. We look forward to introducing more enhancements and updates soon!

Warm regards,

Conflux Documentation Maintenance Team
February 2, 2024

#### Authorizing with Your Github Account to Receive POAP

Visit the [Conflux Document POAP Portal](https://dual-space-nft-frontend.vercel.app). Follow the image instructions to input your Core Space address. Complete the authorization by clicking "one-shot authorize" and then "Do Mint" to obtain your POAP.

<img width="1088" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/a2ed6ef6-e2ab-4269-a01f-92fafa86902e">

> If you encounter a permission issue, please check if you have the mint permission on the card above.

#### Claiming FC

Go to the [Core Space Management Page](https://dual-space-nft-frontend.vercel.app/coreManage). Connect with the account you used to recieve your POAP. Select the POAP and then click on the "Claim xx FC" button.

<img width="1074" alt="image" src="https://github.com/Conflux-Chain/conflux-documentation/assets/17946284/2936d164-13dd-4dbf-b850-9927e00c87ad">

**Comment by ldcrcyy** [comment]
你好，麻烦问一下，那个改进调查，有名单出来嘛。我填写的好像是国内的邮箱[捂脸]

**Reply by darwintree** [reply]
如果当时没有填github账户，可以发邮件到 build@confluxnetwork.org。我们核实您完成过问卷后给github账户补上铸造权限。

**Reply by ldcrcyy** [reply]
好的，已发
